src/components/Home.js

```diff
@@ -0,0 +1,17 @@
+import '../assets/styles/home.css';
+/* eslint-disable max-len */
+function Home() {
+  return (
```

Comment:
- [ ] Please don't disable linter errors instead fix them manually, It will help you to write more clean and professional code.

Reply:
@thecodechaser  please provide or suggest how to fix that without inserting a lot of p tags

Reply:
@Rebelzob Hi, Did you try shiting the text to the next lines?

Reply:
@Rebelzob shifting long text on the next lines should work. My apologies for late response.

Reply:
Thanks for your response, yeah that worked.